src/solver/variable/economy/bindingConstraints/bindingConstraintsMarginalCost.h

```diff
@@ -154,7 +154,7 @@ class BindingConstMarginCost
             pValuesForTheCurrentYear[numSpace].initializeFromStudy(study);
 
         // Set the associated binding constraint
-        associatedBC_ = &(study.runtime->bindingConstraints[bindConstraintGlobalIndex_]);
+        associatedBC_ = study.bindingConstraints.enabled()[bindConstraintGlobalIndex_];
```

Comment:
Matbe you could compute the list of enabled BCs once, for example after loading them.

Reply:
Yes. I had it in mind but forgot. Thank you

Reply:
The value is now cached